docs/bench-report-vlog.md

```
@@ -0,0 +1,208 @@
+
+---
+
+## Detailed Results
+
+### 1. Write Throughput
+
+Measures wall-clock time for 1000 `put()` calls. vLog mode adds overhead because
+`SsTableBuilder::add()` writes large values to the vLog during flush (not on the
+`put()` path itself, but the memtable fills faster triggering more flushes).
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)